src/Shared/LayoutRenderers/AspNetResponseCookieLayoutRenderer.cs

```diff
@@ -149,6 +302,111 @@ private static IList<SetCookieHeaderValue> GetCookies(HttpResponse response)
                 yield return new KeyValuePair<string, string>(cookieName, cookie.Value.ToString());
             }
         }
+
+        private static IEnumerable<SetCookieHeaderValue> GetCookieVerboseValues(IList<SetCookieHeaderValue> cookies, List<string> cookieNames)
+        {
+            foreach (var needle in cookieNames)
+            {
+                for (int i = 0; i < cookies.Count; ++i)
+                {
+                    var cookie = cookies[i];
+                    var cookieName = cookie.Name.ToString();
+                    if (string.Equals(needle, cookieName, StringComparison.OrdinalIgnoreCase))
+                    {
+                        yield return cookie;
+                    }
+                }
+            }
+        }
+
+        private static IEnumerable<SetCookieHeaderValue> GetCookieVerboseAllValues(IList<SetCookieHeaderValue> cookies, ICollection<string> excludeNames)
+        {
+            bool checkForExclude = excludeNames?.Count > 0;
+            for (int i = 0; i < cookies.Count; ++i)
+            {
+                var cookie = cookies[i];
+                var cookieName = cookie.Name.ToString();
+                if (checkForExclude && excludeNames.Contains(cookieName))
+                    continue;
+
+                yield return cookie;
+            }
+        }
+
+        private void SerializeAllProperties(IEnumerable<SetCookieHeaderValue> verboseCookieValues, StringBuilder builder, LogEventInfo logEvent)
+        {
+            switch (OutputFormat)
+            {
+                case AspNetRequestLayoutOutputFormat.Flat:
+                    SerializeAllPropertiesFlat(verboseCookieValues, builder, logEvent);
+                    break;
+                case AspNetRequestLayoutOutputFormat.JsonArray:
```

Comment:
Please include `case AspNetRequestLayoutOutputFormat.JsonDictionary:`

Reply:
Ok, will work on that request

Reply:
Added